EU4toV2/Source/EU4World/Army/EU4Army.cpp

```diff
@@ -105,11 +106,13 @@ std::optional<int> EU4::EU4Army::getProbabilisticHomeProvince(EU4::REGIMENTCATEG
 	return *randomProvince.begin();
 }
 
-void EU4::EU4Army::blockHomeProvince(const int home)
+void EU4::EU4Army::blockHomeProvince(const int blocked)
 {
 	for (const auto& reg_type : EU4::RegimentCategoryTypes)
 	{
```

Comment:
The modern way is similar to the code that was there before:
```homes.erase(std::remove(homes.begin(), homes.end(), blocked), homes.end());```

The original was just missing the step of resizing the original container. See https://en.cppreference.com/w/cpp/algorithm/remove for documentation.